apps/src/applab/setPropertyDropdown.js

```diff
@@ -354,10 +354,10 @@ export function getInternalPropertyInfo(element, friendlyPropName) {
  * @returns {!Array<string> | function} droplet dropdown array or function
  */
 function getPropertyValueDropdown(param2) {
```

Comment:
This function is tested here: https://github.com/code-dot-org/code-dot-org/blob/87fa3d4710d5badc9d658f6befe71ed2265c072b/apps/test/unit/applab/setPropertyDropdownTest.js#L83

I think it well verifies that we always get a valid array so as long as that passes, this should be good.